docusaurus/docs/how-to-guides/ui-extensions/create-an-extension-point.md

```diff
@@ -174,7 +174,7 @@ export const InstanceToolbar = () => {
 ```tsx
 import { usePluginComponents } from '@grafana/runtime';
 
-// Types for the props (passed as a generic to the hook below)
+// Types for the props (passed as a generic to the following hook)
```

Comment:
nit: but this isn't referring to the code immediately following the statement, so not sure this makes things clearer

Reply:
changed it to "hook in the following code block". This is a nit on a nit: doc style guidance is to avoid "above" and "below" in lieu of "previous" and "following"